processor/src/chiplets/mod.rs

```diff
@@ -1,85 +1,91 @@
-/// - columns 1-17: execution trace of hash co-processor
+/// - columns 1-17: execution trace of hash chiplet
 ///
-/// * Bitwise segment: contains the bitwise trace and selectors *
-/// This segment begins at the end of the hasher segment and fills the next rows of the table for
-/// the `trace_len` of the bitwise coprocessor.
+/// * Bitwise segment: contains the trace and selectors for the bitwise chiplet *
+/// This segment begins at the end of the hasher segment and fills the next rows of the trace for
+/// the `trace_len` of the bitwise chiplet.
 /// - column 0: selector column with values set to ONE
 /// - column 1: selector column with values set to ZERO
-/// - columns 2-15: execution trace of bitwise coprocessor
+/// - columns 2-15: execution trace of bitwise chiplet
 /// - column 16-17: unused columns padded with ZERO
 ///
-/// * Padding segment: unused *
-/// This segment begins at the end of the bitwise segment and fills as many rows in the table as
-/// required to ensure that the next and final segment of the trace, containing the memory, will end
-/// at the full length of the execution trace minus the number of random rows.
+/// * Memory segment: contains the trace and selectors for the memory chiplet *
+/// This segment begins at the end of the bitwise segment and fills the next rows of the trace for
+/// the `trace_len` of the memory chiplet.
 /// - column 0-1: selector columns with values set to ONE
 /// - column 2: selector column with values set to ZERO
-/// - columns 3-17: unused columns padded with ZERO
+/// - columns 3-16: execution trace of memory chiplet
+/// - column 17: unused column padded with ZERO
 ///
-/// * Memory segment: contains the memory trace and selectors *
-/// This segment begins at the end of the padding segment and fills the next rows of the table for
-/// the `trace_len` of the memory coprocessor. When it finishes, the execution trace should have
-/// exactly enough rows remaining for the specified number of random rows. The memory segment is
-/// placed at the end of the auxiliary table to simplify transition constraints.
+/// * Padding segment: unused *
+/// This segment begins at the end of the memory segment and fills the rest of the execution trace
+/// minus the number of random rows. When it finishes, the execution trace should have exactly
+/// enough rows remaining for the specified number of random rows.
 /// - columns 0-2: selector columns with values set to ONE
-/// - columns 3-16: execution trace of memory coprocessor
-/// - column 17: unused column padded with ZERO
+/// - columns 3-17: unused columns padded with ZERO
 ///
-pub struct AuxTable {
-    hasher: Hasher,
-    bitwise: Bitwise,
-    memory: Memory,
-    aux_table_bus: AuxTableBus,
+#[derive(Default)]
+pub struct Chiplets {
+    /// Current clock cycle of the VM.
+    clk: usize,
+    pub(super) hasher: Hasher,
+    pub(super) bitwise: Bitwise,
+    pub(super) memory: Memory,
+    pub(super) bus: ChipletsBus,
```

Comment:
We can probably get rid of `pub(super)` modifier. For now at least for `bus`, but in the future maybe for all of them.

Reply:
Thank you, I meant to do this for all of them.